app/src/main/java/kumamotone/sample/eremotethingssample/MainActivity.kt

```diff
@@ -34,7 +35,7 @@ class MainActivity : Activity() {
         }
     }
 
-    private fun changeLight(words: List<String>) {
+    private suspend fun changeLight(words: List<String>) {
```

Comment:
Now, `sendToRemocon` is a suspending function, so `suspend` modifier is required.

Reply:
Okay 👍